lib/game_status.rb

```diff
@@ -2,32 +2,30 @@
 require './main'
 
 module GameStatus
-  class << self
-
-    def guesses_left?(guesses_left)
-      if guesses_left.zero?
-        game_end
-      end
+  include Output
+  def guesses_left?(guesses_left)
+    if guesses_left.zero?
+      game_end
     end
+  end
 
-    private
-
-    def game_end
-      Output.print_no_guesses
-      Output.print_the_code
-      Output.print_start_again
+  private
 
-      if gets.chomp.downcase.eql? 'y'
-        puts 'too bad, thanks for playing though x)'
-        exit(0)
-      end
-      game_exit
-    end
+  def game_end
+    print_no_guesses
+    print_the_code
+    print_start_again
 
-    def game_exit
-      # system 'clear'
-      puts(Colors.cyan_b('Thanks for playing!'))
+    if gets.chomp.downcase.eql? 'y'
+      puts 'too bad, thanks for playing though x)'
       exit(0)
     end
+    game_exit
+  end
+
+  def game_exit
+    # system 'clear'
+    puts(Colors.cyan_b('Thanks for playing!'))
+    exit(0)
   end
 end
```

Comment:
In a `game_status` class I would expect there to be public methods like `game_over?`. When thinking about game states, try to think about the questions you're interesting in asking. "Is the game over?", "How many guesses are left?" etc. Let these methods answer these questions really simply, usually with just a boolean. Let other methods rely on these game-state methods to handle branching behavior like continuing the game or ending the game.